client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -112,6 +116,14 @@ export default {
   },
 
   computed: {
+    ...mapGetters({
+      recordIDs: 'ui/recordPaginationIds',
+    }),
+
+    recordIdx () {
```

Comment:
I would define this in the store and use a getter for it. 
for example recordNavigationIndex